### ISSUE TITLE
Open a new issue on the Google Issue Tracker

### ISSUE THREAD

**Issue by Akeru** [Gericop/Android-Support-Preference-V7-Fix#16]
The issue initially related to this mess (https://code.google.com/p/android/issues/detail?id=183376) was closed and marked as released. Other users have complained this was not the case but Google dev won't bother look at it anymore. 

Since you have a pretty clear understanding of the issue, would you mind opening a new case as asked in comment 64 so they can release a proper fix once for all ?

**Comment by Gericop** [comment]
I'll try, but honestly I don't think it'd result in anything different. They, for whatever reason, think that v7 is only used if you are developing for such old devices, however in my opinion, support classes are better since you don't have to care about the different API levels' method availability. We will see...

**Comment by Akeru** [comment]
I don't have much hope either but thanks for trying anyway. Once opened, please could you add the issue id here for tracking ?

**Comment by Gericop** [comment]
I filed the issue report with the ID [205161](https://code.google.com/p/android/issues/detail?id=205161). I tried to explain them the problems with the workaround approach, I hope they can understand it and see all the efforts it takes to make it material themed...

**Comment by Gericop** [comment]
Looks like they don't really care (what a surprise!). A month is gone and still no updates on the issue...

**Comment by MGaetan89** [comment]
Good news, we can now submit bug fixes to the Support Libraries: https://plus.google.com/+IanLake/posts/Z4nusZ7sjRW

**Comment by Gericop** [comment]
Yeah, I've seen the post. Might check out the branch and upload fix things sometime.
